### ISSUE TITLE
[BUG] Conformance check for scs-0211-v1 (and v2) is broken

### ISSUE THREAD

**Issue by martinmo** [SovereignCloudStack/standards#662]
- the standard v1 doesn't require a default storage class
- but the script will fail if it doesn't exist (so it already does what we intend to do in v2 #658)

**Comment by martinmo** [comment]
I reviewed and tested the script, unfortunately it appears to be broken in more ways, therefore I broadened the scope of this issue:

* it uses packages not specified in `requirements.txt`
* it doesn't yet check what CSI provider is used, to make an informed decision if the storage is local/ephemeral or not (e.g., using a simple local-path provisionar like `kind` sets up already satisfies the script)
* it requires some sonobuoy result file which isn't mentioned anywhere else

**Comment by fraugabel** [comment]
after recompiling requirements and deleting `gen_sonobuoy` function and function calls i could successfully test my kind cluster with the default `"storageClassName":"standard"` . I will now try to provoke errors to see, if it still needs debugging

**Comment by fraugabel** [comment]
There are problems as soon as errors occur. In the event of an error, the `test-pvc` is not removed, which leads to errors when the test is repeated. So i will have a look into the clean-up-part

**Comment by fraugabel** [comment]
I am back on this issue and wonder, whether i should dig deeper into that problem, meaning (write a cleanup bypass part for cleaning up, when errors occur)  or port it to sonobuoy right a way?

**Comment by mbuechse** [comment]
I'm afraid I can't tell what's happening here. Does this have a (non-) associated PR that I could look at?

**Comment by fraugabel** [comment]
@mbuechse right now there is no PR but a branch you can look at. What i am currently working on (from tomorrow on) is the error handling (the exceptions need to be in the right block to get a usable log)
https://github.com/SovereignCloudStack/standards/tree/issue/662_default_storage_class

**Comment by mbuechse** [comment]
It would be great to have a PR. You can open it as draft.

**Comment by mbuechse** [comment]
What is happening here? Of course, there is a PR, and has been since Sep 9?!
https://github.com/SovereignCloudStack/standards/pull/745

**Comment by anjastrunk** [comment]
> What is happening here? Of course, there is a PR, and has been since Sep 9?! #745

@fraugabel Please update this issue or close it, if all tasks are done!

**Comment by fraugabel** [comment]
the progress can be seen here: https://github.com/SovereignCloudStack/standards/tree/issue/662_default_storage_class

currently waiting on feedback or merge

**Comment by mbuechse** [comment]
closing in favor of #930
